### ISSUE TITLE
Change contact details on OBIS hub

### ISSUE THREAD

**Issue by m-hope** [AtlasOfLivingAustralia/obis-hub#1]
As per email request:

-----Original Message-----
From: Andres.Roubicek@csiro.au [mailto:Andres.Roubicek@csiro.au] 
Sent: Wednesday, 8 July 2015 4:24 PM
To: support@ala.org.au
Subject: [ExternalEmail] Change of contact information for OBIS Australia webpages

Hi,

Tony Rees retired last year from CSIRO. Could you please change the Contact Information page (http://obis.ala.org.au/help/contactObis) to the text below, or let me know who I should ask?

---
## NEW OBISAU CONTACT INFORMATION

Mr Andres Roubicek
CSIRO Oceans and Atmosphere Flagship
GPO Box 1538 
Hobart Tasmania 7001, Australia 

Telephone 
+61 3 6232 5002 (international) 
(03) 6232 5002 (within Australia) 

Facsimile 
+61 3 6232 5000 (international) 
(03) 6232 5000 (within Australia) 

Email 
## Andres.Roubicek@csiro.au

Cheers,
## Andres

Mr Andres Roubicek
Data Services Officer
CSIRO/Oceans & Atmosphere Flagship
Castray Esplanade
Hobart, TAS 7000
Phone: +61 3 6232 5002
Fax: +61 3 6232 5000

**Comment by m-hope** [comment]
The OBIS hub was last deployed on ala-iggy.ala.org.au using the old Maven/Googlecode repositories in March 2014 using a single consolidated hubs-webapps WAR file...

Martin has transferred the code for these hubs to GitHub late last year, but as far as I am aware none of this new code has been tested... What I think this means is that any updates to any of these old hubs (also includes: AMRIN, TEPAPA and possibly OZCAM) will require a fair bit of work to set up and test the deployment from the new GitHub/Nexus repositories and then we can make the changes.

**Comment by ansell** [comment]
Contact details should now be:

```
Dr Dave Watts
CSIRO Oceans and Atmosphere
Information and Data Centre
GPO Box 1538
Hobart Tasmania 7001, Australia
Telephone +61 3 6232 5062
Facsimile +61 3 6232 5000
Email OBISAU@csiro.au
```

**Comment by ansell** [comment]
Fixed by pull request #4
